RectangleClient.java

```diff
@@ -0,0 +1,10 @@
+
+public class RectangleClient {
+	public static void main(String[] args) {
+		
+		Rectangle rect1 = new Rectangle(7,9,10,0);
+		System.out.println("r1: " + rect1);
```

Comment:
I don't think you need to put in the "r1: ", did this print the toString() return?